scripts/notarize.sh

```diff
@@ -0,0 +1,83 @@
+appleid="<name@domain.com>" # Email address used for Apple ID
+password="<secret_2FA_password>" # See apps-specific password https://support.apple.com/en-us/HT204397
+teamid="<WWDRTeamID>" # Team idenitifer (if single developer, then set to developer identifier)
```

Comment:
Could hard-code this:

```suggestion
teamid="XM9GC3SUL2" # Team idenitifer (if single developer, then set to developer identifier)
```